core/src/main/java/uk/ac/bbsrc/tgac/miso/core/data/impl/StorageLocation.java

```diff
@@ -49,7 +49,9 @@
     RACK("Rack", Sets.newHashSet(SHELF), BoxStorageAmount.NONE),
```

Comment:
It might be worth renaming this to something like `STACK_RACK` to further differentiate it from `TRAY_RACK`.

Reply:
I'd rather not call them "stack racks" as this isn't how users would think of them. Stacks are used to describe positions within a rack, but the rack is actually just one solid unit. I'm open to suggestions for other name-changes for racks and/or tray racks